crates/serde/src/ssz.rs

```diff
@@ -0,0 +1,50 @@
+//! Serde functions for encoding SSZ primitives.
```

Comment:
We could consider naming this module `ssz_json` and remove the inner `json` mod.